Jenkinsfile

```diff
@@ -32,9 +32,9 @@ pipeline {
             }
             steps {
                 echo "Deploying, because we are on ${env.BRANCH_NAME}"
-                sh "wget https://github.com/asoorm/tyk-git/releases/download/0.1-alpha/tyk-git"
+                sh "wget https://github.com/TykTechnologies/tyk-sync"
```

Comment:
This needs to reference the tyk-sync binary, as per the original, so that it can be downloaded and executed on the CI/CD server

Reply:
I have uploaded a binary to here: `https://github.com/davegarvey/tyk-jenkins-cicd-sync/releases/download/1.0.0/tyk-sync`